core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.scss

```diff
@@ -299,6 +299,45 @@
                 }
             }
         }
+
+        .ai-content-container {
+            &.ProseMirror-selectednode {
+                background-color: $color-palette-primary-op-20;
+                border: 1px solid $color-palette-primary-300;
+            }
+        }
+
+        .loader-style {
+            display: flex;
+            justify-content: center;
+            flex-direction: column;
+            align-items: center;
+            min-height: 12.5rem;
+            min-width: 25rem;
+            width: 25.5625rem;
+            height: 17.375rem;
+            padding: 0.5rem;
+            border-radius: 0.5rem;
+            border: 1.5px solid $color-palette-gray-400;
+        }
+
+        .p-progress-spinner {
+            border: 5px solid $color-palette-gray-300;
+            border-radius: 50%;
+            border-top: 5px solid $color-palette-primary;
+            width: 2.4rem;
+            height: 2.4rem;
+            animation: spin 1s linear infinite;
+        }
+
+        @keyframes spin {
+            0% {
+                transform: rotate(0deg);
+            }
+            100% {
+                transform: rotate(360deg);
+            }
+        }
```

Comment:
We have a primeng component that do this and we need to use it for consistency: https://primeng.org/progressspinner#basic